src/Akeneo/Pim/Enrichment/.php_cd.php

```diff
@@ -127,8 +127,8 @@
         'Akeneo\Pim\Enrichment\Product\API\Command\UserIntent',
         'Akeneo\Pim\Enrichment\Product\API\MessageBus',
         'Akeneo\Pim\Enrichment\Product\API\Query\GetUserIntentsFromStandardFormat',
-        'Akeneo\Pim\Enrichment\Product\Domain\Model\ViolationCode'
-
+        'Akeneo\Pim\Enrichment\Product\Domain\Model\ViolationCode',
+        'Akeneo\Pim\Enrichment\Product\API\Query\GetProductUuidsQuery',
```

Comment:
I think we can put all API classes

```suggestion
        'Akeneo\Pim\Enrichment\Product\API',
```